packages/server/src/fhir/operations/deploy.ts

```diff
@@ -32,7 +32,6 @@ export async function deployHandler(req: FhirRequest): Promise<FhirResponse> {
   }
 
   try {
-    const code = req.body.code as string | undefined;
```

Comment:
Drive-by fix.  This is the exact same as line 25 above, so this line is redundant and confusing.